benchmark/benchmain/main.go

```diff
@@ -123,7 +123,7 @@ const (
 )
 
 var (
-	allWorkloads              = []string{workloadsUnary, workloadsStreaming, workloadsUnconstrained, workloadsAll}
+	allWorkloads              = []string{workloadsAll}
```

Comment:
Why are we deleting other workloads?

Reply:
My bad. I don't know what I was thinking :)
Reverted.